tribits/examples/TribitsExampleProject/README.md

```diff
@@ -19,16 +19,28 @@ with:
     -DTribitsExProj_TRIBITS_DIR=<tribits-dir> \
     -DTribitsExProj_ENABLE_TESTS=ON \
     -DTribitsExProj_ENABLE_ALL_PACKAGES=ON
+    -DTribitsExProj_ENABLE_SECONDARY_TESTED_CODE=ON \
     -DCMAKE_CXX_COMPILER=g++ \
     <path-to-TribitsExampleProject>
 ```
 
-and then build and test with:
+then build and test with:
 
 ```
   make -j4 && ctest -j4
 ```
 
+and then install:
+
+```
+  make DESTDIR=<path-to-install-dir> install
```

Comment:
Did you confirm this works?  I know this works if the CMake project is using `GNUInstallDirs.cmake` and specifying `install()` commands correctly but I am not sure if TriBITS is doing that yet.  See #411.

Reply:
Works fine for my, that's what I did on my machine.

Reply:
Great!  But I thinks that only works if you specify relative paths for the various install dirs fir includes, libs, etc.   But that is the default so that is why this is working I guess.

Reply:
[Here some more info about DESTDIR](https://www.gnu.org/prep/standards/html_node/DESTDIR.html)

Reply:
@bartlettroscoe This is just a sample project, one doesn't have to use the `DESTDIR`, but it gives more control of where one wants to install it. I suppose when one checks it, then build sample app on it, it will be removed from the system anyway.

Reply:
What is wrong with using `-D CMAKE_INSTALL_PREFIX=<install-prefix>`?  That does what people want (i.e. installs headers under `<install-prefix>/include`, libraries under `<install-prefix>/libs`, etc.) and has worked for years.  The other option is to use the newer `cmake --install <build-dir> --prefix <install-prefix>` command supported with CMake 3.15+.  Have you tried that?  That has the advantage that it works with any build tool (i.e. Makefiles or Ninja).

Reply:
Ross, there is nothing wrong with it. It was just a proposal. It could stay as it was.

Reply:
`DESTDIR` is mentioned [here](https://cmake.org/cmake/help/v3.22/variable/CMAKE_INSTALL_PREFIX.html) as well. I will change it as you wanted.

Reply:
`DESTDIR` is obviously an old Makefiles standard and CMake supports it to be a drop-in replacement for other old Makefile-based build systems.  But that does not mean that you have to use it or encourage that people use it.  It seems the purpose is to allow people to test the installation of things that should go under /usr/local without actually doing it.  That is **never** that case with most of the software we are installing.

Let's never mention or use `DESTDIR` in TriBITS documentation or processes.

Reply:
@marcinwrobel1986, more careful reading of:

* https://www.gnu.org/prep/standards/html_node/DESTDIR.html

make it clear that `DESTDIR` is just used for "staging installs" and not for the actual final install location itself.  I got confirmation from Kitware that this is how this is used.  It seems the [CMake documentation for DESTDIR](https://cmake.org/cmake/help/v3.22/envvar/DESTDIR.html#envvar:DESTDIR) and "Professional CMake: 10th edition" don't really mention how `DESTDIR` is supposed to be used (that it is **not** for the final install destination).  That variable name DESTDIR is a terrible name because it does not explain the intent of the variable.  Based on its intent, it should be called something like `STAGING_PREFIX` (see section "The Most Important Naming Consideration" in Chapter 11 "The Power of Variable Names" in the book "Code Complete: 2nd Edition").